### PR TITLE
[Doc Feature][New Teams client - GA][3334706]

### PR DESCRIPTION
**Pull request by v-ypalikila**
updated

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/resources/teams-updates.md](https://github.com/MicrosoftDocs/msteams-docs/blob/9937548ab952c3d8bb80263aa1408d00da32dc0a/msteams-platform%2Fresources%2Fteams-updates.md) | 93 | 4 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/484c6d2d-7a93-44ae-b5b2-57ab9a82e2e7) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/resources/teams-updates.md](https://github.com/MicrosoftDocs/msteams-docs/blob/65558eb307e6e4726f4a20acb6585157573fc222/msteams-platform%2Fresources%2Fteams-updates.md) | 93 | 4 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/c5976181-293c-4580-acc8-6fa26a7e2edb) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/resources/teams-updates.md](https://github.com/MicrosoftDocs/msteams-docs/blob/29cf64b96000bf223e4ca3447ad9182e093fa161/msteams-platform%2Fresources%2Fteams-updates.md) | 95 | 3 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/b0cbb034-ef72-44ba-a30b-a47cf1e8df78) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/mstdd-landing.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/9fdc7654de3f3442d5a0bbb5c9d31264604a5e4c/msteams-platform%2Fmstdd-landing.yml) | 89 | 1 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/0328879e-cc48-4b13-a8a4-207c9cdeea24) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/resources/teams-updates.md](https://github.com/MicrosoftDocs/msteams-docs/blob/9fdc7654de3f3442d5a0bbb5c9d31264604a5e4c/msteams-platform%2Fresources%2Fteams-updates.md) | 95 | 3 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/e07e7657-f61c-4187-8330-be261f4e5f98) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/whats-new.md](https://github.com/MicrosoftDocs/msteams-docs/blob/9fdc7654de3f3442d5a0bbb5c9d31264604a5e4c/msteams-platform%2Fwhats-new.md) | 94 | 38 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/eb312b70-b011-49b4-aca5-79a665a84e61) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/mstdd-landing.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/4bd20bb4aa96271e765de2646801ec4768a34ab5/msteams-platform%2Fmstdd-landing.yml) | 89 | 1 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/9d1f64ec-5185-4f36-880b-38853a80979f) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/resources/teams-updates.md](https://github.com/MicrosoftDocs/msteams-docs/blob/4bd20bb4aa96271e765de2646801ec4768a34ab5/msteams-platform%2Fresources%2Fteams-updates.md) | 92 | 4 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/1dd9e748-f6e3-48d3-b0db-7d4ea38e38d7) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/whats-new.md](https://github.com/MicrosoftDocs/msteams-docs/blob/4bd20bb4aa96271e765de2646801ec4768a34ab5/msteams-platform%2Fwhats-new.md) | 94 | 38 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/b0a3b081-9c48-4646-9293-ce742d2eb45f) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by ydogandjiev — DISMISSED** [review]

File: msteams-platform/resources/teams-updates.md [hunk 1]
```diff
@@ -30,9 +30,7 @@ The Classic Teams client is represented using the `teams` field and the new Team
 
 To ensure a smooth transition, a phased rollout of the new platform is planned as follows:
 
-* **Developer preview**: The new Teams client is available in [public developer preview](dev-preview/developer-preview-intro.md). You can access the new platform and test your apps. We encourage you to provide feedback to help refine the platform.
-
-* **Availability of all platform features from Classic Teams**: All apps are available in the new Teams Client. All platform features except the features listed under [known issues](#known-issues) are now available. Platform features under known issues will be available by September 2023.
+* **Availability of all platform features from Classic Teams**: All apps are available in the new Teams Client. All platform features except the features listed under [known issues](#known-issues) are now available. Platform features under known issues will be available by October 2023.
```
Comment (ChetanSharma-msft) [inline msteams-platform/resources/teams-updates.md:33]: Please verify this date: by October 2023.

**Review by ChetanSharma-msft — CHANGES_REQUESTED** [review]
Added small comment

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/mstdd-landing.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/ea61bcea43ac767e6b5d8fe8e84dfd22369fd1d9/msteams-platform%2Fmstdd-landing.yml) | 89 | 1 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/90eebd07-e924-49f1-a7fc-69911720cdbc) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/resources/teams-updates.md](https://github.com/MicrosoftDocs/msteams-docs/blob/ea61bcea43ac767e6b5d8fe8e84dfd22369fd1d9/msteams-platform%2Fresources%2Fteams-updates.md) | 92 | 4 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/60fab9ba-ef3e-451b-a3ec-1348ea276430) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/whats-new.md](https://github.com/MicrosoftDocs/msteams-docs/blob/ea61bcea43ac767e6b5d8fe8e84dfd22369fd1d9/msteams-platform%2Fwhats-new.md) | 94 | 38 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/7270033c-ad34-48ab-ad33-2b608d09069d) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/mstdd-landing.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/d5539ea90ce73e91c6e0ce2bb1fb72bd829b9121/msteams-platform%2Fmstdd-landing.yml) | 89 | 1 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/3852f19a-6782-45b5-8de7-46a32b5d1f10) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/resources/teams-updates.md](https://github.com/MicrosoftDocs/msteams-docs/blob/d5539ea90ce73e91c6e0ce2bb1fb72bd829b9121/msteams-platform%2Fresources%2Fteams-updates.md) | 93 | 4 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/2d7f5a3d-21b2-4ea2-b5a5-02844498bf40) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/whats-new.md](https://github.com/MicrosoftDocs/msteams-docs/blob/d5539ea90ce73e91c6e0ce2bb1fb72bd829b9121/msteams-platform%2Fwhats-new.md) | 94 | 38 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/b722dbbd-2bd5-4141-b463-45798c01e485) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/mstdd-landing.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/4edfe0cc8be32c7f8840db93081f4b1ed559921b/msteams-platform%2Fmstdd-landing.yml) | 89 | 1 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/2c7fbc66-c5e3-47c2-9edc-00329cb7ac21) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/resources/teams-updates.md](https://github.com/MicrosoftDocs/msteams-docs/blob/4edfe0cc8be32c7f8840db93081f4b1ed559921b/msteams-platform%2Fresources%2Fteams-updates.md) | 94 | 3 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/34723e7a-ab7e-494e-a264-d3a5f1cd2edb) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/whats-new.md](https://github.com/MicrosoftDocs/msteams-docs/blob/4edfe0cc8be32c7f8840db93081f4b1ed559921b/msteams-platform%2Fwhats-new.md) | 94 | 38 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/3315c24d-bcc3-40b6-a264-d307c97e242a) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/resources/teams-updates.md (hunk 1 above)
Comment (v-ypalikila) [reply msteams-platform/resources/teams-updates.md]: Updated to soon.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/mstdd-landing.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/362e76b68a1c74692cad6ba924ec442c361868c7/msteams-platform%2Fmstdd-landing.yml) | 89 | 1 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/bc07d6ae-0385-4ae3-b1ca-1c6767e1ef3a) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/resources/teams-updates.md](https://github.com/MicrosoftDocs/msteams-docs/blob/362e76b68a1c74692cad6ba924ec442c361868c7/msteams-platform%2Fresources%2Fteams-updates.md) | 95 | 3 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/8c69c984-1786-425d-b46d-a6e12d191517) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/whats-new.md](https://github.com/MicrosoftDocs/msteams-docs/blob/362e76b68a1c74692cad6ba924ec442c361868c7/msteams-platform%2Fwhats-new.md) | 94 | 38 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/ca6aace2-4ad5-4b7b-b754-388160b647e9) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by ydogandjiev — APPROVED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/mstdd-landing.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/2705b782a59ed12b942dd482ca4e46fae4b9fd9f/msteams-platform%2Fmstdd-landing.yml) | 89 | 1 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/6dbd3dbf-876b-414d-bd58-dec2ec642f85) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/resources/teams-updates.md](https://github.com/MicrosoftDocs/msteams-docs/blob/2705b782a59ed12b942dd482ca4e46fae4b9fd9f/msteams-platform%2Fresources%2Fteams-updates.md) | 95 | 3 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/176ebec7-3ef5-4fcb-9de8-14709449aa4f) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/whats-new.md](https://github.com/MicrosoftDocs/msteams-docs/blob/2705b782a59ed12b942dd482ca4e46fae4b9fd9f/msteams-platform%2Fwhats-new.md) | 94 | 38 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/6a6d9d46-8278-44e7-ac30-297d1ff58ef5) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/resources/teams-updates.md [hunk 2]
```diff
@@ -30,27 +30,28 @@ The Classic Teams client is represented using the `teams` field and the new Team
...
 
+* Personal static tabs aren't supported in one-on-one conversations with bots in the new Teams client. However, users can open their app from the app bar in Teams and view the personal static tabs.
+
+* [Location](../concepts/device-capabilities/location-capability.md#location-apis) and [Media](../concepts/device-capabilities/media-capabilities.md#media-capability-apis) APIs aren't supported in the new Teams client. We recommend using HTML5 Geolocation, and Media.
+
+* [GetMruTabInstances](/javascript/api/@microsoft/teams-js/pages.tabs?view=msteams-client-js-latest#@microsoft-teams-js-pages-tabs-getmrutabinstances&preserve-view=true) and [GetTabInstances](/javascript/api/@microsoft/teams-js/pages.tabs?view=msteams-client-js-#@microsoft-teams-js-pages-tabs-gettabinstances&preserve-view=true) APIs aren't supported in the new Teams client. We recommend to use the [chat](/graph/api/chat-list-tabs?view=graph-rest-1.0&tabs=http&preserve-view=true) and [channel tabs](/graph/api/channel-list-tabs?view=graph-rest-1.0&tabs=http&preserve-view=true) Graph APIs.
+
```
[the remark this answers is not in the document]
Comment (v-ypalikila) [reply msteams-platform/resources/teams-updates.md:54]: Updated the issue .

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/mstdd-landing.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/fe5bdd425fe431c2cb060c66b116ecd9e57b6729/msteams-platform%2Fmstdd-landing.yml) | 89 | 1 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/d22570f6-9096-4ae0-ba9b-919d384176e8) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/resources/teams-updates.md](https://github.com/MicrosoftDocs/msteams-docs/blob/fe5bdd425fe431c2cb060c66b116ecd9e57b6729/msteams-platform%2Fresources%2Fteams-updates.md) | 95 | 3 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/5bb696ca-4ecb-44d3-ad8a-751b22eb436b) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/whats-new.md](https://github.com/MicrosoftDocs/msteams-docs/blob/fe5bdd425fe431c2cb060c66b116ecd9e57b6729/msteams-platform%2Fwhats-new.md) | 94 | 38 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/f14fb0c9-c897-43f7-900a-9c1463bf1a06) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/mstdd-landing.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/a2fa5a94f5f5bf96d65f6002034552801bc63cb2/msteams-platform%2Fmstdd-landing.yml) | 89 | 1 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/f28bf3d5-9270-45aa-a83b-68ae19c69460) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/resources/teams-updates.md](https://github.com/MicrosoftDocs/msteams-docs/blob/a2fa5a94f5f5bf96d65f6002034552801bc63cb2/msteams-platform%2Fresources%2Fteams-updates.md) | 88 | 8 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/7c711164-b5f1-41c2-a0f6-10deae24cce9) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/whats-new.md](https://github.com/MicrosoftDocs/msteams-docs/blob/a2fa5a94f5f5bf96d65f6002034552801bc63cb2/msteams-platform%2Fwhats-new.md) | 94 | 38 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/b27a8665-b471-4706-a5da-47617846bbb2) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/mstdd-landing.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/e414f4c3e383da954be5fab9f72c9974347228d4/msteams-platform%2Fmstdd-landing.yml) | 89 | 1 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/b5cea408-74f8-4cec-8039-7db4673381c2) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/resources/teams-updates.md](https://github.com/MicrosoftDocs/msteams-docs/blob/e414f4c3e383da954be5fab9f72c9974347228d4/msteams-platform%2Fresources%2Fteams-updates.md) | 93 | 4 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/83e94b2f-8b71-4ae2-a8be-20da8ce2bb65) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/whats-new.md](https://github.com/MicrosoftDocs/msteams-docs/blob/e414f4c3e383da954be5fab9f72c9974347228d4/msteams-platform%2Fwhats-new.md) | 94 | 38 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/b7cb98c6-2365-40b2-8226-b577797e4f66) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by surbhigupta12 — APPROVED** [review]

**Pull request merged**